rest_framework_simple_api_key/admin.py

```diff
@@ -0,0 +1,47 @@
+import typing
+
+from django.contrib import admin, messages
+from django.http.request import HttpRequest
+
+from .models import APIKey
+
+
+class ApiKeyAdmin(admin.ModelAdmin):
+    list_display = ("entity", "revoked", "expires_at")
+
+    list_filter = ("entity", "revoked", "expires_at")
+
+    def get_readonly_fields(
+        self, request: HttpRequest, obj: APIKey = None
+    ) -> typing.Tuple[str, ...]:
+        fields = ()
+
+        if obj and obj.revoked:
+            fields += ("name", "revoked", "expires_at")
```

Comment:
Shouldn't we also include the entity field?

Reply:
Yes